asv/benchmarks.py

```diff
@@ -67,10 +65,13 @@ def __init__(self, conf, benchmarks, regex=None):
                         ', '.join(param_set))
                     if not regex or any(re.search(reg, name) for reg in regex):
                         self[benchmark['name']] = benchmark
-                        self._benchmark_selection[benchmark['name']].append(idx)
+                        bms = self._benchmark_selection[benchmark['name']]
+                        bms.append(idx)
```

Comment:
I'd do. Particularly in this case, you're creating an alias of this list, which makes the code significantly harder to read IMHO.

```suggestion
                        (self._benchmark_selection[benchmark['name']]
                             .append(idx))
```

Reply:
Totally agree from your point of view

Reply:
This is pending. Note that you can change this manually in your local copy and commit. But you can also change it from the GitHub UI, in the commit dropdown to "Commit suggestion".